spec/spec_helper.rb

```diff
@@ -1,2 +1,4 @@
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.